src/settings-ui/Settings.UI/Flyout/AppsListPage.xaml

```diff
@@ -11,19 +11,25 @@
     xmlns:viewmodels="using:Microsoft.PowerToys.Settings.UI.ViewModels"
     mc:Ignorable="d">
 
-    <Grid Background="{ThemeResource LayerOnAcrylicFillColorDefaultBrush}">
+    <Grid Background="{ThemeResource LayerOnAcrylicFillColorDefaultBrush}"
+          Padding="0,12,0,0"
+          RowSpacing="12">
         <Grid.RowDefinitions>
             <RowDefinition Height="Auto" />
             <RowDefinition Height="*" />
         </Grid.RowDefinitions>
-        <Grid Padding="24,32,24,0">
```

Comment:
@Jay-o-Way The margins you mentioned from the design specs are for apps (we need to look at that across the PT apps btw!). But the flyout is replicating a "shell" flyout (like Action Center and/or Start). Different paddings are used there. You can see the Windows UI Library (in Figma) for more details.

Reply:
I'll have another look (when I'm back home).